### PR TITLE
Add reference to Oven

### LINKED ISSUE #4687: Add link to cakedc/oven on installation page.
Newbies won't find it if we don't promote it :)

### PR DESCRIPTION
**Pull request by narendravaghela**
Fixes #4687

### CODE REVIEW

File: en/installation.rst [hunk 1]
```diff
@@ -138,6 +138,25 @@ Be aware that is not recommended, as your application can break when the next ma
 version is being released. Additionally composer does not cache development
 branches, so it slows down consecutive composer installs/updates.
 
+Installation using Oven
+--------------------------------------------------
+
+The another quick way to install CakePHP is `Oven <https://github.com/CakeDC/oven>`_.
```
Comment (ravage84) [inline en/installation.rst]: "Another quick" or "The other quick".

File: en/installation.rst [hunk 2]
```diff
@@ -138,6 +138,25 @@ Be aware that is not recommended, as your application can break when the next ma
 version is being released. Additionally composer does not cache development
 branches, so it slows down consecutive composer installs/updates.
 
+Installation using Oven
+--------------------------------------------------
+
+The another quick way to install CakePHP is `Oven <https://github.com/CakeDC/oven>`_. 
+It is a simple PHP script which installs the CakePHP application skeleton, checks the
```
Comment (ravage84) [inline en/installation.rst]: Aren't the system requirements checked first?

File: en/installation.rst [hunk 3]
```diff
@@ -138,6 +138,25 @@ Be aware that is not recommended, as your application can break when the next ma
...
+necessary system requirements and setup the development environment.
+
+* Upload the oven.php to your server directory.
+* Execute the oven.php via your browser and click on the big oven icon.
+
+Your CakePHP application is ready to go!
```
Comment (ravage84) [inline en/installation.rst]: "After the installation completes, your ..."

File: en/installation.rst [hunk 4]
```diff
@@ -138,6 +138,25 @@ Be aware that is not recommended, as your application can break when the next ma
...
+
+.. note::
+
+    IMPORTANT: This is not a deployment script, it's aimed to help developers installing
```
Comment (ravage84) [inline en/installation.rst]: "script. it is aimed"

Comment (narendravaghela) [inline en/installation.rst]: It was copied from https://github.com/cakedc/oven :)
Anyways, I have updated it.

File: en/installation.rst (hunk 3 above)
Comment (narendravaghela) [inline en/installation.rst]: Done.

File: en/installation.rst (hunk 2 above)
Comment (narendravaghela) [inline en/installation.rst]: Updated the order.

File: en/installation.rst (hunk 1 above)
Comment (narendravaghela) [inline en/installation.rst]: Updated!

**Comment by narendravaghela** [comment]
@ravage84 Could you please review the updates?

**Comment by narendravaghela** [comment]
Thanks 👍

File: en/installation.rst [hunk 5]
```diff
@@ -138,6 +138,25 @@ Be aware that is not recommended, as your application can break when the next ma
...
 
+Installation using Oven
+-----------------------
+
+Another quick way to install CakePHP is `Oven <https://github.com/CakeDC/oven>`_. 
+It is a simple PHP script which checks the necessary system requirements,
+installs the CakePHP application skeleton and sets the development environment up.
+
+* Upload the oven.php to your server directory.
+* Execute the oven.php via your browser and click on the big oven icon.
```
Comment (dereuromark) [inline en/installation.rst]: > Navigate to `/oven.php` via browser ...

maybe?

Comment (narendravaghela) [inline en/installation.rst]: We can use the same steps as https://github.com/CakeDC/oven/blob/master/README.md

Comment (dereuromark) [inline en/installation.rst]: Why duplicating if we can Link to it.

Comment (narendravaghela) [inline en/installation.rst]: What about something like this - 
* Navigate to `/path/to/oven.php` via your browser e.g. `http://localhost/cake/oven.php` and click on the big oven icon.

or just

* Navigate to `/path/to/oven.php` via your browser and click on the big oven icon.

Comment (dereuromark) [inline en/installation.rst]: I mean we already linked to it, I dont see the need to replicate the README instructions here.
We can omit this part, and start again with "After the installation completes".

Comment (narendravaghela) [inline en/installation.rst]: Agree 👍

**Pull request merged**